tests/stdlib/trstgen.nim

```diff
@@ -1034,7 +1034,8 @@ Test1
       """
     var warnings8 = new seq[string]
     let output8 = input8.toHtml(warnings=warnings8)
-    check(warnings8[] == @["input(4, 1) Warning: unknown substitution " &
+    # TODO: the line 1 is arbitrary because reference lines are not preserved
```

Comment:
is that a regression or was `4` meaningless/wrong? (i know that line numbers were often wrong in nim rst2html)

Reply:
not a regression: it's wrong in both cases. Before it was always last line of text, after it's always first line. The problem is that line/column info has never been preserved in any references :-(. We need to add it to `(P)RstNode`, a good task for one of future PRs.